protos/feast/serving/ServingService.proto

```diff
@@ -107,35 +102,40 @@ message GetOnlineFeaturesResponse {
         // Map of feature or entity name to feature/entity statuses/metadata.
         map<string, FieldStatus> statuses = 2;
     }
-   
-    enum FieldStatus {
-        // Status is unset for this field.
-        INVALID = 0;
-
-        // Field value is present for this field and age is within max age.
-        PRESENT = 1;
-
-        // Values could be found for entity key and age is within max age, but
-        // this field value is assigned a value on ingestion into feast.
-        NULL_VALUE = 2;
-
-        // Entity key did not return any values as they do not exist in Feast.
-        // This could suggest that the feature values have not yet been ingested
-        // into feast or the ingestion failed.
-        NOT_FOUND = 3;
-
-        // Values could be found for entity key, but field values are outside the maximum 
-        // allowable range.
-        OUTSIDE_MAX_AGE = 4;
+}
+
+message GetOnlineFeaturesResponseV2 {
+    GetOnlineFeaturesResponseMetadata metadata = 1;
+    repeated FeatureVector results = 2;
```

Comment:
maybe add a comment here that the expectation is that the featurelist length in the metadata should match the number of feature vectors?